.github/workflows/label-analysis.yml

```
@@ -113,3 +113,24 @@ jobs:
           echo "No issue with 'qa/done' and 'qa/no-code-change' labels"
         env:
           LABELS: ${{ needs.fetch-labels.outputs.LABELS}}
+  agenttelemetry-list-change-ack-check:
+    if: github.triggering_actor != 'dd-devflow[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          fetch-depth: 0
+      - name: Setup python
+        uses: actions/setup-python@f677139bbe7f9c59b41e40162b753c062f5d49a3 # v5.2.0
+        with:
+          python-version: 3.11
+          cache: 'pip'
+          cache-dependency-path: '**/requirements*.txt'
+      - name: Install dependencies
+        run: pip install -r requirements.txt -r tasks/requirements.txt
+      - name: Check agent telemetry metric list
+        env:
+          BRANCH_NAME: ${{ github.head_ref }}
+          PR_ID: ${{ github.event.pull_request.number }}
+        run: inv -e github.agenttelemetry-list-change-ack-check
```

It prevents you from collecting it in the task and allows running locally on any PR
```suggestion
        run: inv -e github.agenttelemetry-list-change-ack-check -p $PR_ID
```

---

Will switch to the suggested pattern. Why we have different ways to do the same things (I have borrowed a wrong pattern :) ).

---

This is a good question, probably lacking some control on review to have something uniform